teku/src/main/java/tech/pegasys/teku/cli/options/BeaconNodeDataOptions.java

```diff
@@ -76,6 +76,16 @@ public class BeaconNodeDataOptions extends ValidatorClientDataOptions {
   private long statePruningIntervalSeconds =
```

Comment:
I know this isn't part of this PR but in our recent call you mentioned the limit being in minutes, not seconds. Just checking that this is actually what we want.

Reply:
Yeah, I thought it was defined in minutes when we had the conversation but it's actually following the convention used for the other pruner which is seconds. Could easily change that if we think it makes sense but it would be a little confusing for users to have the state pruner using a different unit for the interval.

Reply:
seconds is ok, but hten we'd want a sanity check. i certainly wouldnt want this happening more often than every slot (ish) for example - i'd potentially set a min of 30 seconds and a max of a day or something...

Reply:
added sanity check for this